src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/perform_interaction_request.cc

```diff
@@ -269,7 +276,16 @@ void PerformInteractionRequest::on_event(const event_engine::Event& event) {
 
   if (!HasHMIResponsesToWait()) {
     LOG4CXX_DEBUG(logger_, "Send response in BOTH iteraction mode");
-    SendBothModeResponse(msg_param);
+    const bool send_vr_params_only =
+        (first_responser_ == FirstAnsweredInterface::VR &&
```

Comment:
@YarikMamykin Swap comparable. Everywhere

Reply:
@AByzhynar fixed in https://github.com/smartdevicelink/sdl_core/pull/3122/commits/8b1b9a79752b6e208df76048cff2e8c307b71492